modules/images/fetchpriority/load.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Module Name: Fetchpriority
+ * Description: Adds `fetchpriority=high` parameter to the non lazy loaded images.
```

Comment:
This description is a bit too technical and too close to the detailed implementation. Also, we can't use Markdown since this will be displayed as is in HTML. I suggest something like:
```suggestion
 * Description: Adds a fetchpriority hint for the primary content image on the page to load faster.
```